UI/src/view/PointAssetLayer.js

```diff
@@ -62,11 +62,23 @@
 
     this.selectControl = selectControl;
 
+    function isAllowedToDrag(features) {
+      if (selectedAsset.exists()) {
```

Comment:
This function could be simplified by : 
` if(selectedAsset.exists() && (layerName == 'trafficSigns' && editConstrains(selectedAsset)))
		return [];
		
	 return features	`